content/sponsors/leaving-github-sponsors/disabling-your-account.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Disabling your GitHub Sponsors account
+intro: 'If you no longer want to be in {% data variables.product.prodname_sponsors %}, you can leave the {% data variables.product.prodname_sponsors %} program.'
+redirect_from:
+  - /articles/leaving-github-sponsors
+  - /github/supporting-the-open-source-community-with-github-sponsors/leaving-github-sponsors
+  - /articles/disabling-your-github-sponsors-account
+  - /github/supporting-the-open-source-community-with-github-sponsors/disabling-your-github-sponsors-account
+versions:
+  free-pro-team: '*'
+permissions: You can easily leave the GitHub Sponsors program if you no longer want to participate. If some people are sponsoring you, you will need to contact GitHub Support about that.
```

Comment:
For the redirects in line 8: Since this is a new article, we don't actually need redirects. We use redirects when change an article title. But I understand why you'd include this here to copy similar patterns found in other articles

Reply:
> For the redirects in line 8: Since this is a new article, we don't actually need redirects. We use redirects when change an article title. But I understand why you'd include this here to copy similar patterns found in other articles

Well, I actually kept them for that reason, but I'm not really sure if we should remove them. Maybe we should remove them according to what you said. 🙂

Reply:
I removed them in the commits I made below. 💫